src/components/drawing/index.js

```diff
@@ -579,16 +579,17 @@ drawing.makeTester = function(gd) {
         tester.node()._cache = {};
     }
 
-    gd._tester = tester;
-    gd._testref = testref;
+    drawing.tester = gd._tester = tester;
+    drawing.testref = gd._testref = testref;
```

Comment:
re: `Drawing.` vs `gd.` - we had better pick one and go use only that. As long as we look into this issue in the near future I don't care too much which we use. As is, if there *is* any difference between them, we'll be making a mess of it as soon as we have plots in two different windows... but in an order-dependent way that will be very hard to debug.